.github/workflows/ci.yml

```diff
@@ -111,7 +111,17 @@ jobs:
       - uses: julia-actions/julia-buildpkg@v1
         env:
           PYTHON: ""
-      - uses: julia-actions/julia-runtest@v1
+      - name: Run tests without coverage
+        uses: julia-actions/julia-runtest@v1
+        with:
+          coverage: false
+        env:
+          PYTHON: ""
+          TRIXI_TEST: ${{ matrix.trixi_test }}
+      - name: Run tests with coverage
+        uses: julia-actions/julia-runtest@v1
+        with:
+          coverage: true
```

Comment:
I'll just add this as a comment here to help us remember it - we do have some tests that do not really take long with coverage (`mpi`) or even run without coverage (`threaded`). Right now, these tests are duplicated with this setup.

Reply:
Would it make sense to add a special argument `coverage_only` that the defaults to `false`?  Or, more general, an argument like `mode` that can be `coverage`, `result` or `both`, with the latter being the default?

Reply:
I like that idea